Framework/DataHandling/inc/MantidDataHandling/LoadILLIndirect2.h

```diff
@@ -53,15 +53,15 @@ class DLLExport LoadILLIndirect2 : public API::IFileLoader<Kernel::NexusDescript
   std::string m_instrumentName; ///< Name of the instrument
 
   // Variables describing the data in the detector
-  size_t m_numberOfTubes{16};          // number of tubes - X
```

Comment:
I don't see them in the parameters files, I assume they are set based on the data. But in that case I don't see much benefit, and actually having base defaults seems safer, in case something is broken in the file.

Reply:
Then I propose to keep those default values but move them to the initializer list of the constructor, so all values are in one file, instead of being split between the two source files.